scripts/export_comments.py

```diff
@@ -62,24 +62,31 @@ def create_comments_excel_file(survey_id, submissions):
 
     for submission in submissions:
         comment = submission.data['data'].get('146')
+
+        boxes_selected = ""
+        for key in ['146a', '146b', '146c', '146d', '146e', '146f', '146h']:
+            if key in submission.data['data'].keys():
+                boxes_selected = boxes_selected + ' ' + key
```

Comment:
Petty but to avoid a leading space before the first selected box should we have one after the last? Visually a little better. I'm sure there is a better way again but this is simplest.
`boxes_selected = boxes_selected + key + ' '`

Reply:
Agreed, much better.  Fixed